README.md

```diff
@@ -1,12 +1,224 @@
 # Project eAST
 
 ## Overview
-Project eAST (eA simulation toolkit) is led by Makoto Asai (SLAC) and builds on the work done in existing detector simulations for the EIC. To ease leveraging new and rapidly evolving computing technologies, we are implementing a common and integrated approach for fast and full detector simulations in Geant4 with a plug and play modular approach.
-
-## Execute eAST
-To execute eAST, go to Core directory and follow its [README](Core/README.md).
+Project *eAST* (eA Simulation Toolkit) is led by Makoto Asai (SLAC) and builds on the work done in existing detector simulations for the EIC. To ease leveraging new and rapidly evolving computing technologies, we are implementing a common and integrated approach for fast and full detector simulations in Geant4 with a plug and play modular approach.
 
 ## References
 * [Overview Presentation](https://docs.google.com/presentation/d/1i3_MG26J93OqOuZx8MJY_btmpGpuxPRCOfdJHAHFPwY/edit?usp=sharing)
 * [Proposal](https://docs.google.com/document/d/1-EduKk_hCUr2lnKZFyqCMyzQBaM4ABRTRfFeOrEtWo8/edit?usp=sharing)
 
+## Installing eAST
+
+*eAST* runs on top of the latest public version of Geant4 (currently [version 10.7.p02](https://geant4.web.cern.ch/support/download)). You need to install Geant4 in advance to installing eAST with the following options. Please refer to [the Geant4 installation guide](https://geant4-userdoc.web.cern.ch/UsersGuides/InstallationGuide/html/index.html).
+- GEANT4_BUILD_MULTITHREADED
+- GEANT4_USE_GDML (it requires _Xerces-C++_ pre-installed)
+- GEANT4_USE_OPENGL_X11 (if you use event display. it requires _X11_ and _OpenGL_ libraries)
+- GEANT4_USE_QT (if you use Qt window. it requires _Qt5_ and _OpenGL_ libraries)
+
+Building is done via cmake, using the `CMakeLists.txt` file at the top level. It is recommended to use a separate build directory. Please make sure all the necessary Geant4 environment variables are set in advance.
+```
+cd east
+mkdir build
+cd build
+cmake ..
+make
```

Comment:
I think more modern cmake is invoked as
```
cmake -Bbuild -S.
cmake --build build
```

Reply:
I'm seeing it more and more often, but it's still somewhat unusual. Also, I personally find it gimicky. But we can change it if  you prefer!